config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1263 @@
+    parent: "agent_kubernetes"
+    weight: 307
+  - name: "Custom Metrics & DogStatsD"
+    url: "agent/kubernetes/dogstatsd/"
+    parent: "agent_kubernetes"
+    weight: 308
+  - name: "Legacy Versions"
+    url: "agent/kubernetes/legacy/"
+    parent: "agent_kubernetes"
+    weight: 309
+
+  ## AGENT - Autodiscovery
+
+  - name: "Autodiscovery"
+    url: "agent/autodiscovery/"
+    identifier: "agent_autodiscovery"
+    parent: "agent"
+    weight: 4
+  - name: "Integrations Templates"
+    url: "agent/autodiscovery/integrations/"
+    parent: "agent_autodiscovery"
+    weight: 401
+  - name: "Container Identifier"
+    url: "agent/autodiscovery/ad_identifiers/"
+    parent: "agent_autodiscovery"
+    weight: 402
+  - name: "Template Variables"
+    url: "agent/autodiscovery/template_variables/"
+    parent: "agent_autodiscovery"
+    weight: 403
+  - name: "Tag Extraction"
+    url: "agent/autodiscovery/tag/"
+    parent: "agent_autodiscovery"
+    weight: 404
+  - name: "Discovery Management"
+    url: "agent/autodiscovery/management/"
+    parent: "agent_autodiscovery"
+    weight: 405
+  - name: "Troubleshooting"
+    url: "agent/autodiscovery/troubleshooting/"
+    parent: "agent_autodiscovery"
+    weight: 406
+  - name: "Cluster Checks"
+    url: "agent/autodiscovery/clusterchecks/"
+    parent: "agent_autodiscovery"
+    weight: 407
+
+  ## AGENT - LOGS
+
+  - name: "Log Collection"
+    url: "agent/logs/"
+    identifier: "agent_logs"
+    parent: "agent"
+    weight: 5
+  - name: "Advanced Log Collection"
+    url: "agent/logs/advanced_log_collection"
+    parent: "agent_logs"
+    weight: 501
+  - name: "Proxy"
+    url: "agent/logs/proxy"
+    parent: "agent_logs"
+    weight: 502
+
+  - name: "Proxy"
+    url: "agent/proxy/"
+    parent: "agent"
+    weight: 6
+
+  - name: "Upgrade to Agent v6"
+    url: "agent/guide/upgrade-to-agent-v6/"
+    parent: "agent"
+    weight: 7
+
+  ## AGENT - Troubleshooting
+  - name: "Troubleshooting"
+    url: "agent/troubleshooting/"
+    parent: "agent"
+    weight: 8
+    identifier: "agent_troubleshooting"
+  - name: "Debug Mode"
+    url: "agent/troubleshooting/debug_mode/"
+    parent: "agent_troubleshooting"
+    weight: 801
+  - name: "Agent Flare"
+    url: "agent/troubleshooting/send_a_flare/"
+    parent: "agent_troubleshooting"
+    weight: 802
+  - name: "Agent Check Status"
+    url: "agent/troubleshooting/agent_check_status/"
+    parent: "agent_troubleshooting"
+    weight: 803
+  - name: "NTP Issues"
+    url: "agent/troubleshooting/ntp/"
+    parent: "agent_troubleshooting"
+    weight: 804
+  - name: "Permission Issues"
+    url: "agent/troubleshooting/permissions/"
+    parent: "agent_troubleshooting"
+    weight: 805
+  - name: "Integrations Issues"
+    url: "agent/troubleshooting/integrations/"
+    parent: "agent_troubleshooting"
+    weight: 806
+
+  ## AGENT - Guides
+  - name: "Guides"
+    url: "agent/guide/"
+    parent: "agent"
+    weight: 9
```

Comment:
Guides doesn't show in the nav. Check the preview.

Reply:
fixed